Lesson15/lesson15-ezgi/script.js

```diff
@@ -98,23 +98,23 @@ const string2Exercise13 =
 	'Bundle deal: Save $50 when you buy two products together!';
 const percentSymbol = '%';
 
-console.log('Exercise13: ');
-console.log('Exercise13: ');
+console.log('Exercise13: ',string1Exercise13.includes(percentSymbol));
+console.log('Exercise13: ',string2Exercise13.includes(percentSymbol));
 console.log('------------------------');
 // 14. Use the `substring` method to extract the word "Databases" from string stringExercise14
 // based on two indices and print it to the console.
 const stringExercise14 =
 	'Databases are an essential component of many applications.';
-
-console.log('Exercise 14: ');
+	const result2 = stringExercise14.substring(0,8);
+	console.log('Exercise 14: ' , result2);
 console.log('------------------------');
 // 15. Remove extra spaces from the string stringExercise15 and print the result to the console.
 const stringExercise15 = '    Hello world    ';
-
-console.log('Exercise15: ');
+const result3 = stringExercise15.trim();
+console.log('Exercise15: ' , result3);
 console.log('------------------------');
 // 16. Extract the price and currency from a string below using "slice" method and print it to console.
 const planPriceString = 'Premium plan - 9.99 USD/month.'; // Should return "9.99 USD"
-
-console.log('Exercise16: ');
+const planPriceStringIndexOf = planPriceString.indexOf("9.99 USD");
+console.log('Exercise16: ' , planPriceString.slice(planPriceStringIndexOf,planPriceStringIndexOf+8));
```

Comment:
Well done with the start index! For the end index, it would be good to use a dynamic value as well. So, instead of `planPriceStringIndexOf+8` you could do:
`planPriceStringIndexOf + "9.99 USD".length` OR `planPriceString.indexOf("/")` if '/' symbol always separates the currency and period.